activiti-cloud-service-common/activiti-cloud-services-test/pom.xml

```diff
@@ -118,4 +123,65 @@
       <artifactId>activiti-cloud-services-test-security-keycloak</artifactId>
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <version>${build-helper-maven-plugin.version}</version>
+        <executions>
+          <execution>
+            <id>add-generated-assertions-folder</id>
+            <phase>generate-test-sources</phase>
+            <goals>
+              <goal>add-test-source</goal>
+            </goals>
+            <configuration>
+              <sources>
+                <source>${generated-assertions-folder}</source>
+              </sources>
+            </configuration>
+          </execution>
+        </executions>
```

Comment:
You can remove this block and keep only `groupId` and `artifactId` as the other configuration elements (including the version) are inherited from the [cloud-build](https://github.com/Activiti/activiti-cloud/blob/develop/activiti-cloud-build/pom.xml#L328-L346) that has it inside `pluginManagement` section.

Reply:
Done. Thanks!